seovoc/.htaccess

```diff
@@ -0,0 +1,49 @@
+# Turn off MultiViews
+Options -MultiViews
+
+# Directive to ensure *.rdf files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+AddType application/n-triples .n3
+AddType application/ld+json .jsonld
+# Rewrite engine setup
+RewriteEngine On
+
+# Rewrite rule to serve HTML content from the vocabulary URI if requested
+# RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
+# RewriteCond %{HTTP_ACCEPT} text/html [OR]
+# RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+# RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+# RewriteRule ^([^/]+)$ https://example.org/index.html [R=303,L]
+
+Rewrite rule to serve JSON-LD content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} application/ld\+json
+RewriteRule ^([^/]+)$ https://raw.githubusercontent.com/wordlift/wl-ontology/main/seovoc.jsonld [R=303,L]
+
+# Rewrite rule to serve RDF/XML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
+# Temporary redirect until we finalize repo and file naming
+RewriteRule ^([^/]+)$ https://raw.githubusercontent.com/wordlift/wl-ontology/main/SEOntology.owl [R=307,L]
+
+# Rewrite rule to serve N-Triples content from the vocabulary URI if requested
+# RewriteCond %{HTTP_ACCEPT} application/n-triples
+# RewriteRule ^([^/]+)$ https://example.org/ontology.nt [R=303,L]
+
+Rewrite rule to serve TTL content from the vocabulary URI if requested
```

Comment:
You need to comment out this line, or it won't work

Reply:
Oh, sorry my mistake

Reply:
It's done, I am not sure if I should resolve the conversation or not :)